src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: transform 0.3s, color 0.3s;
```

Comment:
`0.3s` values are used in several places, so it makes sense to create a variable for this value.
Let's do this!

Reply:
okay